### PR TITLE
qc login

### PR DESCRIPTION
**Pull request by velrest**
- fix(docker): run server with poetry so deps are found
- feat(hous_stat_creds): creds are identified by user and x-camac-group

### CODE REVIEW

File: ebau_gwr/token_proxy/migrations/0003_housingstatcreds_group.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Generated by Django 3.2.13 on 2022-07-04 16:47
+
+from django.db import migrations, models
+
+
+def deleteCredentials(apps, schema_editor):
```
Comment (winged) [inline ebau_gwr/token_proxy/migrations/0003_housingstatcreds_group.py:6]: pep8

**Review by winged — CHANGES_REQUESTED** [review]
Naming functions: use pep8 naming (snake case), otherwise LGTM

File: ebau_gwr/token_proxy/migrations/0003_housingstatcreds_group.py (hunk 1 above)
Comment (velrest) [reply ebau_gwr/token_proxy/migrations/0003_housingstatcreds_group.py]: My bad, renamed to snake case

**Comment by fugal-dy** [comment]
looking good!

**Review by fugal-dy — APPROVED** [review]

**Pull request merged**